### PR TITLE
DTSPRB-567: Adding in annotations to allow/disallow for null values

### PR DESCRIPTION
**Pull request by SarahLittlejohn**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-567

### Change description ###
Fixing master branch 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

**Review by sanjayparekh-hmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java [hunk 1]
```diff
@@ -36,6 +37,8 @@ public BigDecimal getApplicationFee(BigDecimal amountInPound) {
             return BigDecimal.ZERO;
         }
 
+        Objects.nonNull(responseEntity);
+
```
Comment (isha2040) [inline src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java]: this new line could be removed

File: src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java [hunk 2]
```diff
@@ -52,6 +55,8 @@ public BigDecimal getCopiesFee(Long copies) {
...
```
Comment (isha2040) [inline src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java]: this new line could be removed

File: src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java (hunk 1 above)
Comment (isha2040) [inline src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java]: this new line could be removed

File: src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java (hunk 2 above)
Comment (isha2040) [inline src/main/java/uk/gov/hmcts/probate/service/fee/FeeService.java]: this new line could be removed

**Pull request merged**